.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: "build-test"
```

Comment:
This comes from the TS action example before, it is just the action running itself to check it works.